site/content/en/docs/Installation/terraform.md

```diff
@@ -134,3 +135,73 @@ terraform destroy
 
 ### Reference
 Details on how you can authenticate your AKS terraform provider using official [instructions](https://www.terraform.io/docs/providers/azurerm/auth/service_principal_client_secret.html)
+
+
+{{% feature publishVersion="1.3.0" %}}
+# Installing the Agones as Terraform submodule on AWS EKS
+
+{{< alert title="Note" color="info" >}}
+AWS EKS Terraform configuration is not fully functional.
+`terraform destroy` leaves EKS cluster, so please remove EKS cluster and other objects, like VPC Security Groups manually.
+[Pull requests](https://github.com/googleforgames/agones/pulls) to finish the functionality are appreciated.
+{{< /alert >}}
+
+You can use Terraform to provision your Amazon EKS (Elastic Kubernetes Service) cluster and install Agones on it using Helm Terraform provider.
+
+The example of EKS submodule config file could be found here:
+ {{< ghlink href="examples/terraform-submodules/eks/module.tf" >}}Terraform configuration with Agones submodule{{< /ghlink >}}
+
+Copy `module.tf` file into a separate folder.
+
+Configure your AWS CLI tool [CLI configure](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html):
+```
+aws configure
+```
+
+Configure your terraform:
+```
+terraform init
+```
+
+By editing `modules.tf` you can change some parameters as you need, for example EC2 instance type. Note that the maximum number of instances in the workers group is limited by 3.
```

Comment:
```suggestion
By editing `modules.tf` you can change the parameters that you need to, such as the EC2 instance type. Note that the maximum number of instances in the workers group is limited by 3.
```

> Note that the maximum number of instances in the workers group is limited by 3.

What does that mean? I can't more than 3 instances?

Reply:
at the moment of writing that was true, I copied this from the docs of terraform provider. Let me check this statement.

Reply:
Suggestion on the sentence structure.

> Note that the maximum number of instances in the workers group is limited **to** 3.

I wasn't sure what "limited by 3" meant